### PR TITLE
Add Restream

### PR DESCRIPTION
**Pull request by achudinovskyh**

### CODE REVIEW

File: _data/entertainment.yml [hunk 1]
```diff
@@ -139,6 +139,13 @@ websites:
     facebook: qobuz
     img: qobuz.png
 
+  - name: Restream.io
+    url: http://restream.io/
+    img: restream.png
+    tfa:
+      - totp
+    doc: https://support.restream.io/en/articles/870429-two-factor-authentication
```
Comment (phallobst) [inline _data/entertainment.yml]:
```suggestion
    doc: https://support.restream.io/en/articles/870429
```

**Review by phallobst — CHANGES_REQUESTED** [review]
My impression is that this should be in the "Social" category rather than "Entertainment".

**Comment by phallobst** [comment]
This icon looks a bit nicer, I think:
restream

**Comment by achudinovskyh** [comment]
both requests were addressed

**Review by phallobst — APPROVED** [review]

**Review by dsoegijono — APPROVED** [review]

File: _data/social.yml [hunk 2]
```diff
@@ -225,6 +225,13 @@ websites:
       - totp
     doc: https://www.reddithelp.com/hc/en-us/articles/360043470031
 
+  - name: Restream.io
+    url: http://restream.io/
```
Comment (dsoegijono) [inline _data/social.yml]:
```suggestion
    url: https://restream.io/
```

Prefer https

**Review by dsoegijono — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by dsoegijono — APPROVED** [review]

**Pull request merged**